microovn/ovn/environment.go

```diff
@@ -177,7 +177,7 @@ func cleanupPaths() error {
 		errs = append(
 			errs,
 			fmt.Errorf(
-				"failed to create backup directory '%s'. Refusing to continue with data removal: %s",
+				"failed to create backup directory '%s'. refusing to continue with data removal: %s",
```

Comment:
nit: same as mentioned above. Either retain capitalization of "inner" sentence, or turn this error to single sentence with comma instead of dot.